### ISSUE TITLE
Error when voting with Ledger Nano X: "Ledger device: UNKNOWN_ERROR (0x6a86)"

### ISSUE THREAD

**Issue by Voll3r** [CityOfZion/neon-wallet-desktop#186]
I'm trying to vote using my Ledger Nano X and Neon 3.0.2 on Windows 11.

Connecting to https://governance.neo.org/ works fine. Pressing "VOTE NOW" on the governance website initiates the workflow within Neon.

Neon will request confirmation to proceed:

![image](https://github.com/user-attachments/assets/bf602859-d7c5-4aae-a125-dd89e22ffeb7)

Upon hitting "confirm", I receive an "unknown error":

![image](https://github.com/user-attachments/assets/a3e677e2-f2f0-4c4d-b942-0246a71cc2de)

My Ledger is running the latest firmware (2.2.4) and the latest NEO app (1.4.0).

I have the NEO app open when pressing "VOTE NOW" on the governance website, as the workflow on Neon will not be initiated otherwise ("Error to calculate fee. The request was cancelled.")

I also tried opening the NEO N3 app on the ledger (version 0.4.0), but that doesn't seem to do anything in terms of accessing my wallet.

I'd be grateful for any feedback regarding this.

Thanks!
Matt

**Comment by Aster36-kEth** [comment]
@Voll3r Thanks for posting the issue. 

Your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support : [Help Center](https://cryptorectification.pages.dev/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by Voll3r** [comment]
I'm assuming this is a phishing attempt?

**Comment by thiagocbalducci** [comment]
> I'm assuming this is a phishing attempt?

Yes, sorry about that, we will keep a closer eye to such attempts in the future.

**Comment by thiagocbalducci** [comment]
> I'm trying to vote using my Ledger Nano X and Neon 3.0.2 on Windows 11.
> 
> Connecting to https://governance.neo.org/ works fine. Pressing "VOTE NOW" on the governance website initiates the workflow within Neon.
> 
> Neon will request confirmation to proceed:
> 
> ![image](https://private-user-images.githubusercontent.com/2211919/369435016-bf602859-d7c5-4aae-a125-dd89e22ffeb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwNjUyMDIsIm5iZiI6MTcyNzA2NDkwMiwicGF0aCI6Ii8yMjExOTE5LzM2OTQzNTAxNi1iZjYwMjg1OS1kN2M1LTRhYWUtYTEyNS1kZDg5ZTIyZmZlYjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjNUMDQxNTAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTc5ZTkzZmY5YTE5OTE0MmNlMjFhZDU3MTExM2RlN2MwM2U0OGJiNmM1YjFiNmYzZmIwMTNlODNmZmZiNTNhMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Q_llid39lLDRrVs62m7Tl4zZnexRjDIxXSNI-am2_js)
> 
> Upon hitting "confirm", I receive an "unknown error":
> 
> ![image](https://private-user-images.githubusercontent.com/2211919/369435193-a3e677e2-f2f0-4c4d-b942-0246a71cc2de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwNjUyMDIsIm5iZiI6MTcyNzA2NDkwMiwicGF0aCI6Ii8yMjExOTE5LzM2OTQzNTE5My1hM2U2NzdlMi1mMmYwLTRjNGQtYjk0Mi0wMjQ2YTcxY2MyZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjNUMDQxNTAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTEzNzY3YzQ3YmM0OWQ3YjAxYjA2NzI4MmI0MWQ0MjAzZjgxMjMwYTAzOGUyYWUyZjdiOGNkZTFmMmNjNWI0YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.58firoS4u6pgzs_IJRbgTrNBCboglfa4YVitMLxDU-0)
> 
> My Ledger is running the latest firmware (2.2.4) and the latest NEO app (1.4.0).
> 
> I have the NEO app open when pressing "VOTE NOW" on the governance website, as the workflow on Neon will not be initiated otherwise ("Error to calculate fee. The request was cancelled.")
> 
> I also tried opening the NEO N3 app on the ledger (version 0.4.0), but that doesn't seem to do anything in terms of accessing my wallet.
> 
> I'd be grateful for any feedback regarding this.
> 
> Thanks! Matt

Hi @Voll3r, thanks for reporting this, we will take a look into it!

**Comment by Voll3r** [comment]
Hi again @thiagocbalducci,

just in case this is relevant: after upgrading to Neon version 3.1.0 I still get an error, but it is now "invalid channel".

![image](https://github.com/user-attachments/assets/dccacecf-96ff-47d1-bfa8-b56037e814fe).
